app/templates/service.html

```diff
@@ -32,13 +32,22 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     {% if service_unavailability_information.type == 'framework_update' %}
+    {# TODO: Hardcoding this banner for G11 for now, should be derived from the framework object  #}
+       {% if service.frameworkName == 'G-Cloud 11' %}
+           {{ dmBanner({
+        "title": "This G-Cloud 11 service has been suspended and is no longer searchable on the Digital Marketplace.",
+        "text": "The G-Cloud 11 framework expires on 18 December 2020 to allow in progress procurements to be concluded. Existing contracts with {} are still valid.".format(
+            service.supplierName)
+      }) }}
+   {% else %}
       {{ dmBanner({
         "title": "This {} service is no longer available to buy.".format(service.frameworkName),
         "text": "The {} framework expired on {}. Any existing contracts with {} are still valid.".format(
             service.frameworkName,
             framework_expires_at_date,
             service.supplierName)
       }) }}
+   {% endif %}
```

Comment:
Is this indentation correct? It doesn't seem to line up with the `if` to me...

Reply:
it is correct

Reply:
OK I don't know, I got intellij to indent it and committed that in 49bc491